components/generic_modal.scss

```diff
@@ -216,6 +216,31 @@
                     }
                 }
 
+                &.delete {
+                    @include primary-button;
+                    @include button-medium;
```

Comment:
this is already applied in line 196 ... is there a specific reason this is done again?